satpy/tests/test_modifiers.py

```diff
@@ -392,6 +392,104 @@ def test_compositor(self, calculator, apply_modifier_info, sza):
+        assert ray_cor.attrs['wavelength'] == wavelength
+        assert ray_cor.attrs['resolution'] == resolution
+        assert ray_cor.attrs['atmosphere'] == atmosphere
+        assert ray_cor.attrs['aerosol_type'] == aerosol_type
+        assert ray_cor.attrs['lim_low'] == lim_low
+        assert ray_cor.attrs['lim_high'] == lim_high
+        assert ray_cor.attrs['strength'] == strength
+
+        area, dnb = self.make_data_area()
+        c01 = xr.DataArray(dnb,
+                           dims=('y', 'x'),
+                           attrs={
+                               'platform_name': 'GOES-16',
+                               'calibration': 'reflectance', 'units': '%', 'wavelength': (0.45, 0.47, 0.49),
+                               'name': 'C01', 'resolution': 1000, 'sensor': 'abi',
+                               'start_time': '2017-09-20 17:30:40.800000', 'end_time': '2017-09-20 17:41:17.500000',
+                               'area': area, 'ancillary_variables': [],
+                               'orbital_parameters': {
+                                   'satellite_nominal_longitude': -89.5,
+                                   'satellite_nominal_latitude': 0.0,
+                                   'satellite_nominal_altitude': 35786023.4375,
+                               },
+                           })
+
+        c02 = xr.DataArray(dnb,
+                           dims=('y', 'x'),
+                           attrs={
+                               'platform_name': 'GOES-16',
+                               'calibration': 'reflectance', 'units': '%', 'wavelength': (0.59, 0.64, 0.69),
+                               'name': 'C02', 'resolution': 500, 'sensor': 'abi',
+                               'start_time': '2017-09-20 17:30:40.800000', 'end_time': '2017-09-20 17:41:17.500000',
+                               'area': area, 'ancillary_variables': [],
+                               'orbital_parameters': {
+                                   'satellite_nominal_longitude': -89.5,
+                                   'satellite_nominal_latitude': 0.0,
+                                   'satellite_nominal_altitude': 35786023.4375,
+                               },
+                           })
+
+        res = ray_cor([c01, c02])
+
+        assert isinstance(res, xr.DataArray)
+        assert isinstance(res.data, da.Array)
+        assert res.attrs['platform_name'] == 'GOES-16'
+        assert res.attrs['calibration'] == 'reflectance'
+        assert res.attrs['units'] == '%'
+        assert res.attrs['wavelength'] == wavelength
+        assert res.attrs['name'] == name
+        assert res.attrs['resolution'] == resolution
+        assert res.attrs['sensor'] == 'abi'
+        assert res.attrs['start_time'] == '2017-09-20 17:30:40.800000'
+        assert res.attrs['end_time'] == '2017-09-20 17:41:17.500000'
+        assert res.attrs['area'] == area
+        assert res.attrs['ancillary_variables'] == []
```

Comment:
I don't think we need to check all of these attributes unless we assume they are going to be broken. This seems mostly like clutter. I know some of the other tests do stuff like this, but I disagree with those checks there too.